### ISSUE TITLE
`async` object initialization using shorthand notation causes NRE

### ISSUE THREAD

**Issue by GilianJuice** [trullock/NUglify#138]
In some cases I am getting NullReferenceExceptions while using NUglify. After some trial and error I found the problems that cause these. I am using version 1.9.1,

The first one is when you try to minify this code:
```javascript
var test = (p) => {
    console.log("test", p);
};

test.foo = {
    bar: 1,
    async testSub() {
        console.log("testSub");
    }
};
```

If you remove the "async" before "testSub", the minify will work without exceptions (but obiously my code won't work then, since I'm using await in that function in my original code).

This will cause the following error:

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at NUglify.JavaScript.JSParser.ParseArrowFunction(AstNode parameters)
   at NUglify.JavaScript.JSParser.ParseLeftHandSideExpression(Boolean isMinus)
   at NUglify.JavaScript.JSParser.ParseUnaryExpression(Boolean& isLeftHandSideExpr, Boolean isMinus)
   at NUglify.JavaScript.JSParser.ParseExpressionStatement(Boolean fSourceElement)
   at NUglify.JavaScript.JSParser.ParseStatement(Boolean fSourceElement, Boolean skipImportantComment)
   at NUglify.JavaScript.JSParser.ParseStatements(BlockStatement block)
   at NUglify.JavaScript.JSParser.InternalParse()
   at NUglify.JavaScript.JSParser.Parse(DocumentContext sourceContext)
   at NUglify.Uglify.Js(String source, String fileName, CodeSettings codeSettings)
   at NUglify.Uglify.Js(String source, CodeSettings codeSettings)
```

The other issue can be found here: https://github.com/trullock/NUglify/issues/139

**Comment by trullock** [comment]
Curious. Would you mind dividing this issue into two, one for each case as they're almost certainly different. I'll take a look tonight

**Comment by GilianJuice** [comment]
Sure, I have removed one issue from this one and made a new one for it: https://github.com/trullock/NUglify/issues/139

**Comment by trullock** [comment]
I suspect this is to do with the shorthand syntax.as a possible workaround try using traditional object initialisation syntax.

I'll take a look at this next

**Comment by trullock** [comment]
Fixed in 1.9.2

**Comment by GilianJuice** [comment]
Thanks! I have tested this and it works.
